docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -319,8 +313,8 @@ The Alluxio master writes journal to the PersistentVolume defined in
 *alluxio-master-journal-pv.yaml.template*.
 - *multiMaster-EmbeddedJournal* directory gives you the Kubernetes ConfigMap, 3 Alluxio masters and
 a set of Alluxio workers.
-The Alluxio masters each write to its `alluxio-journal-volume`, which is an `emptyDir` that gets
-wiped out when the Pod is shut down.
+The Alluxio masters each write to its `alluxio-journal-alluxio-master-{index}` volume,
```

Comment:
the claim name feels weird. can we change it?

Reply:
This claim name is auto-generated with `{volume name}-{pod name}` i believe. There should be a way to configure this tho. Do you have a naming patter in mind?

When my use I wasn't very concerned with the name, the PVC is managed by the StatefulSet so all I do is creating my PVs and check the PVC status with `kubectl get pvc`.

Reply:
Correction: The name of this generated PVC is `{metadata.name}-{pod name}`